### ISSUE TITLE
Passing extra parameters like language_override

### ISSUE THREAD

**Issue by allaire** [intercom/intercom-wordpress#27]
Hi,

We're using WPML and we would like to be able to set extra parameters (perhaps a JSON in the admin settings?)

I currently edited the plugin to add the following:

```
array("app_id" => get_option('intercom')['app_id'], "language_override" => ICL_LANGUAGE_CODE)
```

**Comment by bobjflong** [comment]
Hi @allaire I think we are trying to keep the configuration options as small as possible. However I do like the idea of supporting well known constants that make sense to include. Here's a PR for that if you'd like to take a look https://github.com/intercom/intercom-wordpress/pull/29

**Comment by allaire** [comment]
@bobjflong It looks good. Does it works well in case `ICL_LANGUAGE_CODE` is not defined?

Thanks!

**Comment by bobjflong** [comment]
@allaire if `ICL_LANGUAGE_CODE` is not defined then `language_override` won't be present in the snippet yep :)

**Comment by allaire** [comment]
Great! Then I think you can release the kraken!

**Comment by bobjflong** [comment]
@allaire released as `2.1.0`, let me know if that works out!
